Be able to configure hostname and multiline prompt

CDA0 opened the pull request:
* Added an option to have a multiline prompt
* Added an option to remove the hostname

CDA0 committed:
* Added an option to remove the hostname
* Added an option to have a multiline prompt

bluz71 commented on command_prompt.bash:
```diff
@@ -181,7 +183,7 @@ _seafly_command_prompt() {
     local prompt_start
     if [[ $SEAFLY_SHOW_USER = 1 ]]; then
         prompt_start="\[$SEAFLY_HOST_COLOR\]\u@\h"
-    else
+    elif [[ $SEAFLY_SHOW_HOST = 1 ]]; then
         prompt_start="\[$SEAFLY_HOST_COLOR\]\h"
     fi
```
This logic is not correct, I believe it should be something like

```sh
if [[ $SEAFLY_SHOW_USER = 1 && $SEAFLY_SHOW_HOST = 1 ]]; then
         prompt_start="\[$SEAFLY_HOST_COLOR\]\u@\h"
elif [[ $SEAFLY_SHOW_USER = 1 ]]; then
         prompt_start="\[$SEAFLY_HOST_COLOR\]\u"
elif [[ $SEAFLY_SHOW_HOST = 1 ]]; then
        prompt_start="\[$SEAFLY_HOST_COLOR\]\h"
fi
```

bluz71 commented on command_prompt.bash:
```diff
@@ -199,7 +201,14 @@ _seafly_command_prompt() {
+    else
+        prompt_end="\[\${_seafly_colors[\$((!\$?))]}\] $SEAFLY_PROMPT_SYMBOL\[\$NOCOLOR\] "
+    fi
+    unset _seafly_git
```
Please delete the `unset _seafly_git` at line 211.

bluz71 commented on command_prompt.bash:
```diff
@@ -199,7 +201,14 @@ _seafly_command_prompt() {
-    local prompt_end="\[\${_seafly_colors[\$((!\$?))]}\] $SEAFLY_PROMPT_SYMBOL\[\$NOCOLOR\] "
+    local prompt_end
+    if [[ $SEAFLY_MULTILINE = 1 ]]; then
+        prompt_start="\n$prompt_start"
```
The `prompt_start` change on line 206 is better placed up where prompt_start is set, just after all the `SHOW_USER` & `SHOW_HOST` logic is done.

```sh
if [[ $SEAFLY_MULTILINE = 1 ]]; then
    prompt_start="\n$prompt_start"
fi
```

That way all the `prompt_start`, `prompt_middle` and `prompt_end` logic is clustered together.

bluz71 commented:
The aim of this pull request is within project scope, so yes we will merge this after some minor fixes. Once multiline is set the result does look a little like the [Zsh Pure prompt](https://github.com/sindresorhus/pure).

After the review items in the `command_prompt.bash` file are done I will merge.

There are some changes in the README Mardown file I will take care of after merging, so don't spend any time there.

Best regards.

CDA0 committed:
fixes based on comments on PR

CDA0 commented:
I have (hopefully) addressed the comments.

I added the first multiline carriage return to the prefix rather than start, hope that is correct.

I also noticed alacritty needed to be added alongside kitty, hope you don't mind me squeezing that in here too.

bluz71 commented:
Cool.

I will review these tomorrow and merge.
